### PR TITLE
[homekit] Support multiple values per enum mapping

### LINKED ISSUE #15672: [homekit] support multiple Enum Values for current state of accessory
Hi,

right now we can define for accessory characteristic current state only one Enum Value, for example thermostat current heating cooling mode HEAT(1), COOL(2), etc...
I have many devices which report HEAT(1), SUPER HEAT(3), COOL(2) SUPER COOL(4), in this case I want to show thermostat current state HEAT[1,3 ]and COOL[2,4]. I think it can be use as one of. 

Thanks
Grzegorz

### PR DESCRIPTION
**Pull request by ccutrer**
Closes #15672

### CODE REVIEW

File: bundles/org.openhab.io.homekit/README.md [hunk 1]
```diff
@@ -724,6 +724,21 @@ All enum values can be customized via item metadata. I.e. `HEAT="heating", COOL=
 They are appropriately marked.
 Enums that are linked to Switches or Contacts have an `inverted` param that will reverse the sense of `ON`/`OFF` or `OPEN`/`CLOSED`.
 
+Enum mappings can have multiple values for a single key.
+These must be an array, not a comma separated string.
```
Comment (lsiepel) [inline bundles/org.openhab.io.homekit/README.md:728]: Filie based configuration should be fully supported. It should be possible to have a comma seperated list in the config. Both file based and UI based config should be able to handle that. 
For reference you can look at [the tr064 binding](https://www.openhab.org/addons/bindings/tr064/#a-note-on-textual-configuration) => `macOnline` property.

Comment (ccutrer) [reply bundles/org.openhab.io.homekit/README.md:728]: I'm [working on it](https://github.com/openhab/openhab-core/pull/4330). Thanks for the reference though, it helps me figure out this xtext thing.

**Comment by kaikreuzer** [comment]
https://github.com/openhab/openhab-core/pull/4330 has been merged!

**Review by lsiepel — APPROVED** [review]
Thanks LGTM

**Pull request merged**